ush/templates/FV3LAM_wflow.xml

```diff
@@ -215,36 +214,6 @@ MODULES_RUN_TASK_FP script.
   </task>
 {% endif %}
 
-<!--
```

Comment:
Combined all the data getting into a single task and left it named as it originally was for "LBCS"